src/bigquery.ts

```diff
@@ -93,7 +93,7 @@ export type Query = JobRequest<bigquery.IJobConfigurationQuery> & {
   types?: string[] | string[][] | {[type: string]: string[]};
   job?: Job;
   maxResults?: number;
-  timeoutMs?: number;
```

Comment:
Is this the breaking change part? Do you know the original intention of having `timeoutMs` here? I couldn't find that we handled `timeoutMs` ourselves, so maybe we thought that's what the API called it, so we were just saying that we accept it here, thinking the API would do something with it?

Reply:
Yeah, I was assuming removing this property would be considered breaking. And my theory is that `timeoutMs` ended up here by accident because it is part of a [QueryRequest](https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs/query#QueryRequest), but it was never actually being used for anything by the API since it isn't part of [JobConfigurationQuery](https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationQuery).

And I'm not totally clear on the exact boundaries of what is considered a breaking change in our libraries. Does the fact that job.getQueryResults() will now return an error if a user's timeout has been exceeded, rather than just continuing to check for results regardless of the value provided, count as breaking?

Reply:
> Yeah, I was assuming removing this property would be considered breaking.

TS-type breaks aren't too scary to me, because the developer will have a chance to catch it before anything is running in production.

>  Does the fact that job.getQueryResults() will now return an error if a user's timeout has been exceeded, rather than just continuing to check for results regardless of the value provided, count as breaking?

I would think this is more of a fix, in that now a user providing a timeout will actually get their expected results.

In both of these cases, the affected users should be a pretty low number. However, I still don't mind the 'better safe than sorry' approach, and releasing this under a major bump. ¯\_(ツ)_/¯

Reply:
@stephenplusplus I think you're right that it's actually more of a bug fix and we have a few users who need this corrected, so I think it's fine to go ahead and merge.